fasthttp_test.go

```diff
@@ -378,28 +378,34 @@ func TestFastHTTPNodeApplyMiddleware(t *testing.T) {
 
 	router := NewFastHTTPRouter().(*fastHTTPRouter)
 
-	router.GET("/x/{param}", func(ctx *fasthttp.RequestCtx) {
+	router.GET("/{param}", func(ctx *fasthttp.RequestCtx) {
```

Comment:
When you try to fix it you want to make that test pass, now change the test because it doesn't pass.
This is why I updated that test in my pr to cover that case https://github.com/vardius/gorouter/pull/15